euphony/src/androidTest/java/co/euphony/ui/EuphonyRxPanelTest.kt

```diff
@@ -0,0 +1,114 @@
+package co.euphony.ui
+
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.PixelMap
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createComposeRule
+import co.euphony.common.Constants
+import co.euphony.ui.theme.LightGreen
+import co.euphony.ui.theme.LightRed
+import co.euphony.ui.theme.LightSkyBlue
+import co.euphony.ui.viewmodel.EuphonyRxPanelViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.junit.MockitoJUnitRunner
+
+@ExperimentalMaterialApi
+@ExperimentalComposeUiApi
+@RunWith(MockitoJUnitRunner::class)
+class EuphonyRxPanelTest {
+
+    @get:Rule
+    val composeTestRule = createComposeRule()
+
+    @Mock
+    private lateinit var viewModel: EuphonyRxPanelViewModel
+
+    private val isListening = MutableStateFlow(false)
+    private val isListenStarted = MutableStateFlow(false)
```

Comment:
I think `isListenStarted` is not needed, so we can eliminate this variable :smile: